arch/arm/core/Kconfig

```diff
@@ -26,6 +26,19 @@ config CPU_CORTEX_M
 	help
 	  This option signifies the use of a CPU of the Cortex-M family.
 
+config CPU_CORTEX_R
+	bool
+	# Omit prompt to signify "hidden" option
```

Comment:
I wonder how helpful these comments are in practice. There's a gazillion of them in Zephyr.